### PR TITLE
Add instructions on how to SSH into AWS

### PR DESCRIPTION
**Pull request by issyl0**

### CODE REVIEW

File: source/manual/howto-ssh-to-machines-in-aws.html.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+parent: "/manual.html"
+last_reviewed_on: 2017-12-05
+review_in: 1 month
+---
+
+This document explains how to SSH into machines in AWS, as it's markedly
+different than for Carrenza.
+
+In AWS, there are no static hostnames, so we can't have
+`backend-1.backend.integration` to SSH to like in Carrenza. Instead, EC2
```
Comment (surminus) [inline source/manual/howto-ssh-to-machines-in-aws.html.md]: Remove `Instead`. 

`An EC2 instance...`

File: source/manual/howto-ssh-to-machines-in-aws.html.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+instance has dynamically assigned IPs, which means hostnames like
+`ip-10-1-5-53.eu-west-1.compute.internal`. Luckily, each Puppeted instance has
```
Comment (surminus) [inline source/manual/howto-ssh-to-machines-in-aws.html.md]: It's not lucky :)

**Comment by surminus** [comment]
We could also add a note about being able to proxy straight into specific nodes. I have SSH config similar to:
```
Host *.aws.integration
  ProxyCommand ssh -e none %r@jumpbox.blue.integration.govuk.digital -W $(echo %h | sed 's/\.aws.integration$//'):%p
```

When using Icinga and can get a specific hostname, it's easier to use this proxy method:

```
ssh ip-1-2-3-4.eu-west-1.compute.internal.aws.integration
```

File: source/manual/howto-ssh-to-machines-in-aws.html.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        ip-10-1-5-53.eu-west-1.compute.internal
+        ip-10-1-6-253.eu-west-1.compute.internal
+
+   Choose one, and copy/paste into a normal SSH command:
+
+        ssh ip-10-1-5-53.eu-west-1.compute.internal
+
+4. If your SSH key has forwarded correctly, you will be on the new machine.
+
+5. If you're bored of copy/paste and would like to just get to a single node,
```
Comment (surminus) [inline source/manual/howto-ssh-to-machines-in-aws.html.md]: Remove colloquial language: `If you're bored of copy/paste`. 

You can just say: `To get to a single node, use:`

File: source/manual/howto-ssh-to-machines-in-aws.html.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+   for example:
+
+        ip-10-1-5-53.eu-west-1.compute.internal
+        ip-10-1-6-253.eu-west-1.compute.internal
+
+   Choose one, and copy/paste into a normal SSH command:
+
+        ssh ip-10-1-5-53.eu-west-1.compute.internal
+
+4. If your SSH key has forwarded correctly, you will be on the new machine.
```
Comment (surminus) [inline source/manual/howto-ssh-to-machines-in-aws.html.md]: I don't think this should be a step in it's own right. Let's assume it works.

File: source/manual/howto-ssh-to-machines-in-aws.html.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+5. If you're bored of copy/paste and would like to just get to a single node,
+   you can use the `--single-node` switch on `govuk_node_list`, straight into
+   your SSH command:
+
+        ssh `govuk_node_list -c backend --single-node`
+
+Alternatively, you can SSH into specific nodes directly from your local machine with:
```
Comment (surminus) [inline source/manual/howto-ssh-to-machines-in-aws.html.md]: I would split the methods into their own sections so they're easily linkable. 

This also removes language such as "alternatively".

File: source/manual/howto-ssh-to-machines-in-aws.html.md (hunk 3 above)
Comment (issyl0) [reply source/manual/howto-ssh-to-machines-in-aws.html.md]: But I *was* bored of copy/paste! 😄

File: source/manual/howto-ssh-to-machines-in-aws.html.md (hunk 5 above)
Comment (issyl0) [reply source/manual/howto-ssh-to-machines-in-aws.html.md]: I added sections, though they're not very well worded.

**Pull request merged**